zipline/algorithm.py

```diff
@@ -250,6 +250,12 @@ def __init__(self, *args, **kwargs):
             identifiers : List
                 Any asset identifiers that are not provided in the
                 equities_metadata, but will be traded by this TradingAlgorithm
+            equity_slippage : SlippageModel
```

Comment:
Is there a reason to prefer making this an arg vs. using the existing `set_slippage` API function?  It seems a little confusing to have two different ways to do this.